src/nvim/vim.h

```diff
@@ -338,14 +338,9 @@ enum {
  * functions of these names. The declarations would break if the defines had
  * been seen at that stage.  But it must be before globals.h, where error_ga
  * is declared. */
-#if !defined(FEAT_GUI_W32) && !defined(FEAT_GUI_X11) \
-  && !defined(FEAT_GUI_GTK) && !defined(FEAT_GUI_MAC)
 # define mch_errmsg(str)        fprintf(stderr, "%s", (str))
 # define display_errors()       fflush(stderr)
 # define mch_msg(str)           printf("%s", (str))
-#else
-# define USE_MCH_ERRMSG
```

Comment:
I'm fairly confident now that `USE_MCH_ERRMSG` can be removed.

The one instance in `message.c` used to be for error output in some GUIs when they didn't have their native error output channel ready yet during startup. This then either used `fprintf` to stderr for the message or collected the messages in `garray_T error_ga`. `error_ga` in `globals.h` is the other instance guarded by `USE_MCH_ERRMSG`. I believe @ZyX-I's `stripdecls.py` script was a one time thing, so we can just leave it there.

A quick web search confirms that `USE_MCH_ERRMSG` isn't used outside of the Vim source code.